app/models/character.rb

```diff
@@ -8,8 +8,7 @@ class Character < ActiveRecord::Base
   belongs_to :equipped_weapon, :class_name => "Item"
 
   def hp
-    (items + incidents).reduce(avatar.skill_set.health.to_i) do |acc, item_incident|
-      acc + item_incident.skill_set.health.to_i
+    (items + incidents).reduce(avatar.skill_set.health.to_i) do |acc, item_incident| acc + item_incident.skill_set.health.to_i
```

Comment:
Line is too long. [126/80]<br>Prefer `inject` over `reduce`.<br>Block body expression is on the same line as the block start.